src/appkit.cs

```diff
@@ -12623,6 +12692,11 @@ partial interface NSResponder : NSCoding, NSTouchBarProvider /* radar://42781537
 		[Export ("restorableStateKeyPaths", ArgumentSemantic.Copy)]
 		string [] RestorableStateKeyPaths ();
 
+		[Mac (12,0)]
+		[Static]
+		[Export ("allowedClassesForRestorableStateKeyPath:")]
+		Class[] GetAllowedClassesForRestorableStateKeyPath (string keyPath);
```

Comment:
Wondering if this name could exclude KeyPath since it is the parameter name.
Swift name in the docs is `allowedClasses(forRestorableStateKeyPath:)`
but would `GetAllowedClassesForRestorableState (string keyPath)` work here?

Reply:
thanks! I'll just simplify to `GetAllowedClasses` then - I think RestorableStateKeyPath is one thing, so it could be confusing to imply its for the restorablestate and not specify the keypath